src/main/java/Duke.java

```diff
@@ -58,16 +61,16 @@ private static void handleCommand(String command) throws ArrayIndexOutOfBoundsEx
             terminateProgram();
             break;
         case "done":
```

Comment:
I like how you went back and ensured every variable had spaces in between making them neat